pkg/docker/config/config_test.go

```diff
@@ -394,21 +442,21 @@ func TestGetAuthFailsOnBadInput(t *testing.T) {
 	// remove the invalid config file
 	os.RemoveAll(configPath)
 	// no config file present
-	username, password, err = GetAuthentication(nil, "index.docker.io")
+	auth, err = GetCredentials(nil, "index.docker.io")
 	if err != nil {
 		t.Fatalf("got unexpected error: %#+v", err)
 	}
-	if len(username) > 0 || len(password) > 0 {
-		t.Fatalf("got unexpected not empty username/password: %q/%q", username, password)
+	if len(auth.Username) > 0 || len(auth.Password) > 0 {
```

Comment:
This should also be a `types.DockerAuthConfig{}` comparison I think. (See the other one above).